tests/unit/test_req.py

```diff
@@ -647,14 +671,17 @@ def test_exclusive_environment_markers():
     assert req_set.has_requirement("Django")
 
 
-def test_mismatched_versions(caplog):
+def test_mismatched_versions(caplog: pytest.LogCaptureFixture) -> None:
     req = InstallRequirement(
         req=Requirement("simplewheel==2.0"),
         comes_from=None,
     )
     req.source_dir = "/tmp/somewhere"  # make req believe it has been unpacked
     # Monkeypatch!
-    req._metadata = {"name": "simplewheel", "version": "1.0"}
+    metadata = email.message.Message()
+    metadata["name"] = "simplewheel"
+    metadata["version"] = "1.0"
+    req._metadata = metadata
     req.assert_source_matches_version()
```

Comment:
```suggestion
    metadata = email.message.Message()
    metadata["name"] = "simplewheel"
    metadata["version"] = "1.0"
    req._metadata = metadata

    req.assert_source_matches_version()
```

As I was saying, I like my whitespace.